app/models/contact.rb

```diff
@@ -0,0 +1,39 @@
+# == Schema Information
+#
+# Table name: contacts
+#
+#  id               :bigint           not null, primary key
+#  name             :string           not null
```

Comment:
I suspect again, `name` might need a validation for presence.

Reply:
added!